tamr_unify_client/models/attribute/subattribute.py

```diff
@@ -0,0 +1,45 @@
+class SubAttribute:
+    """
+    An attribute which is itself a property of another attribute.
+    See https://docs.tamr.com/reference#attribute-types
+
+    :param client: Delegate underlying API calls to this client.
+    :type: :class:`~tamr_unify_client.Client`
+    :param data: JSON data representing this attribute
+    :type: :py:class:`dict`
+    """
+
+    def __init__(self, client, data):
```

Comment:
I don't think `client` is necessary here. Usually we need it to fetch the resource itself or any subresources, but this isn't a resource and has no fetchable subresources. Could we remove `client` (including docstring) from this file?